root/static/scripts/work.js

```diff
@@ -1,6 +1,14 @@
+// @flow
```

Comment:
We can do an eslint run here? It picked up quite a few inconsistencies.

Reply:
Opened this PR before I setup eslint :) I will go back and run it on things.